redhat/project.go

```diff
@@ -1,78 +0,0 @@
-package redhat
```

Comment:
Why you removed this?

Reply:
the related distro/project.go are removed because unneeded, in order to de-duplicate code. You'll find the only required stuff in project/project.go file

Reply:
Ook! :+1: